### ISSUE TITLE
Send impl for tty::windows::Console

### ISSUE THREAD

**Issue by hayd** [kkawakam/rustyline#160]
I am seeing the following error on Windows (but not on osx/linux, these both work), here DenoRepl is a struct containing an `Editor<()>`.

```
error[E0277]: `*mut std::ffi::c_void` cannot be sent between threads safely
   --> ../../src\isolate.rs:379:5
    |
379 |     tokio::spawn(task);
    |     ^^^^^^^^^^^^ `*mut std::ffi::c_void` cannot be sent between threads safely
    |
    = help: within `std::collections::HashMap<std::string::String, repl::DenoRepl>`, the trait `std::marker::Send` is not implemented for `*mut std::ffi::c_void`
    = note: required because it appears within the type `rustyline::tty::windows::Console`
    = note: required because it appears within the type `rustyline::Editor<()>`
    = note: required because it appears within the type `repl::DenoRepl`
    = note: required because it appears within the type `(std::string::String, repl::DenoRepl)`
    = note: required because it appears within the type `std::marker::PhantomData<(std::string::String, repl::DenoRepl)>`
    = note: required because it appears within the type `std::collections::hash::table::RawTable<std::string::String, repl::DenoRepl>`
    = note: required because it appears within the type `std::collections::HashMap<std::string::String, repl::DenoRepl>`
    = note: required because of the requirements on the impl of `std::marker::Send` for `std::sync::Mutex<std::collections::HashMap<std::string::String, repl::DenoRepl>>`
    = note: required because it appears within the type `isolate::IsolateState`
    = note: required because of the requirements on the impl of `std::marker::Send` for `std::sync::Arc<isolate::IsolateState>`
    = note: required because it appears within the type `[closure@../../src\isolate.rs:375:17: 378:8 state:std::sync::Arc<isolate::IsolateState>, req_id:i32]`
    = note: required because it appears within the type `futures::future::chain::Chain<std::boxed::Box<dyn futures::Future<Error=errors::DenoError, Item=std::boxed::Box<[u8]>> + std::marker::Send>, futures::FutureResult<(), errors::DenoError>, [closure@../../src\isolate.rs:375:17: 378:8 state:std::sync::Arc<isolate::IsolateState>, req_id:i32]>`
    = note: required because it appears within the type `futures::AndThen<std::boxed::Box<dyn futures::Future<Error=errors::DenoError, Item=std::boxed::Box<[u8]>> + std::marker::Send>, std::result::Result<(), errors::DenoError>, [closure@../../src\isolate.rs:375:17: 378:8 state:std::sync::Arc<isolate::IsolateState>, req_id:i32]>`
    = note: required because it appears within the type `futures::MapErr<futures::AndThen<std::boxed::Box<dyn futures::Future<Error=errors::DenoError, Item=std::boxed::Box<[u8]>> + std::marker::Send>, std::result::Result<(), errors::DenoError>, [closure@../../src\isolate.rs:375:17: 378:8 state:std::sync::Arc<isolate::IsolateState>, req_id:i32]>, [closure@../../src\isolate.rs:378:18: 378:24]>`
    = note: required by `tokio::spawn`
```
https://ci.appveyor.com/project/deno/deno/builds/20021726
I'm not sure why this is. Is it because Console doesn't implement Send?

**Comment by hayd** [comment]
A workaround: https://github.com/piscisaureus/deno/commit/02b8202fe8963e7efa2b527ded8e405b671b2af1
Should this be incorporated into rustyline?

**Comment by hayd** [comment]
I think this is fixed by #161.
